lib/lunchbot_web/live/component/shared_cart/office_selector.ex

```diff
@@ -0,0 +1,12 @@
+defmodule LunchbotWeb.SharedCart.OfficeSelector do
```

Comment:
nit: Would it be possible to have the office selector modules separated out of this commit beforehand? The logic is great but it might be nice for the commit structure

Reply:
@force10269 it is currently the first commit?